connectors/webhook-connector/element-templates/webhook-connector-intermediate.json

```
@@ -0,0 +1,207 @@
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.subtype"
+      }
+    },
+    {
+      "label": "Webhook ID",
+      "type": "String",
+      "group": "endpoint",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.context"
+      },
+      "description": "The webhook ID is a part of the URL"
+    },
+    {
+      "id": "shouldValidateHmac",
+      "label": "HMAC Authentication",
+      "group": "endpoint",
+      "description": "Choose whether HMAC verification is enabled. <a href='https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/http-webhook/#make-your-http-webhook-connector-for-receiving-messages-executable' target='_blank'>See documentation</a> and <a href='https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/http-webhook/#example' target='_blank'>example</a> that explains how to use HMAC-related fields",
+      "value": "disabled",
+      "type": "Dropdown",
+      "choices": [
+        {
+          "name": "Enabled",
+          "value": "enabled"
+        },
+        {
+          "name": "Disabled",
+          "value": "disabled"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.shouldValidateHmac"
+      }
+    },
+    {
+      "label": "HMAC Secret Key",
+      "description": "Shared secret key",
+      "type": "String",
+      "group": "endpoint",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.hmacSecret"
+      },
+      "condition": {
+        "property": "shouldValidateHmac",
+        "equals": "enabled"
+      }
+    },
+    {
+      "label": "HMAC Header",
+      "description": "Name of header attribute that will contain the HMAC value",
+      "type": "String",
+      "group": "endpoint",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.hmacHeader"
+      },
+      "condition": {
+        "property": "shouldValidateHmac",
+        "equals": "enabled"
+      }
+    },
+    {
+      "label": "HMAC Algorithm",
+      "group": "endpoint",
+      "description": "Choose HMAC algorithm",
+      "value": "sha_256",
+      "type": "Dropdown",
+      "choices": [
+        {
+          "name": "SHA-1",
+          "value": "sha_1"
+        },
+        {
+          "name": "SHA-256",
+          "value": "sha_256"
+        },
+        {
+          "name": "SHA-512",
+          "value": "sha_512"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.hmacAlgorithm"
+      },
+      "condition": {
+        "property": "shouldValidateHmac",
+        "equals": "enabled"
+      }
+    },
+    {
+      "label": "Correlation key (process)",
+      "type": "String",
+      "group": "activation",
+      "feel": "optional",
+      "description": "Sets up the correlation key from process variables.",
```

I believe, we don't put dots at the end of descriptions

```suggestion
      "description": "Sets up the correlation key from process variables",
```